packages/connector-feishu-web/src/constant.ts

```diff
@@ -0,0 +1,28 @@
+import type { ConnectorMetadata } from '@logto/connector-kit';
+import { ConnectorPlatform } from '@logto/connector-kit';
+
+export const codeEndpoint = 'https://passport.feishu.cn/suite/passport/oauth/authorize';
+
+export const accessTokenEndpoint = 'https://passport.feishu.cn/suite/passport/oauth/token';
+
+export const userInfoEndpoint = 'https://passport.feishu.cn/suite/passport/oauth/userinfo';
+
+export const defaultMetadata: ConnectorMetadata = {
+  id: 'feishu-web',
+  target: 'feishu',
+  platform: ConnectorPlatform.Web,
+  name: {
+    en: 'Feishu',
+    'zh-CN': '飞书',
+    'tr-TR': 'Feishu',
+    ko: 'Feishu',
```

Comment:
These two lines can be saved as well, as the name will fallback to English if a local translation is not found